spec/features/availability_spec.rb

```diff
@@ -36,17 +36,4 @@
       expect(page.all('.location--online .umlaut .fulltext .response_item').length).to be >= 4
     end
   end
-
```

Comment:
The record that was tested here no longer has a temp location (not on reserve anymore). I don't know how to stub the Ajax call.